src/absil/il.fs

```diff
@@ -1552,7 +1557,7 @@ and [<Sealed>] ILTypeDefs(f : unit -> (string list * string * ILAttributes * Laz
     let mutable array = InlineDelayInit<_>(f)
     let mutable dict = InlineDelayInit<_>(fun () -> 
             let arr = array.Value
-            let t = Dictionary<_,_>(HashIdentity.Structural)
+            let t = Dictionary<_,_>(3, HashIdentity.Structural)
```

Comment:
Maybe, but that's not the default value for this contructor. In any case, this will probably go away, as it's just a work around a deficiency in Fable (can't do comparers), when it's resolved I'll take it out.